doc/developer/diagnostic-queries.md

```diff
@@ -127,3 +127,47 @@ where
 group by mdo.id, mdo.name
 order by sum(mas.records) desc;
 ```
+
+### How can I check whether work is distributed equally across workers?
+
+Work is distributed across workers by the hash of their keys. Thus, work can
+become skewed if situations arise where Materialize needs to use arrangements
+with very few or no keys. Example situations include:
+* views that maintain order by/limit/offset
+* cross joins
+* joins where the join columns have very few unique values.
+
+Currently, only one worker ingests a partition, so there could be ingest skew if
+some partitions have more records than others.
+
+```sql
+-- average the total time spent by each operator across all workers
+create view avg_elapsed_by_id as
+select
+    id,
+    avg(elapsed_ns) as avg_ns
+from
+    mz_scheduling_elapsed
+group by
+    id;
+
+-- get operators where one worker has spent more than 2 times the average
+-- amount of time spent. The number 2 can be changed according to the threshold
+-- for the amount of skew deemed problematic.
```

Comment:
Nit, but inconsistent capitalization and punctuation across the comments here. I'm a fan of capitals and punctuation when it is a sentence, and not otherwise (which is what the rest of the doc uses, I think/hope).